proto/viam/app/data/v1/data.proto

```diff
@@ -376,3 +379,12 @@ message BoundingBoxLabelsByFilterRequest {
 message BoundingBoxLabelsByFilterResponse {
   repeated string labels = 1;
 }
+
+message GetDatabaseUserRequest {
+  string org_id = 1;
+}
+
+message GetDatabaseUserResponse {
+  string username = 1; // unsure if we need to repond with username because every org is given one default username
+  string connection_string = 2; // hostname...(I think?)
```

Comment:
Another scope change; we should change this to hostname, since the connection string varies slightly based on whatever connection options the user decides at connection time. It can also vary depending on which client they're connecting with. However the hostname is just the DNS hostname/domain name of the server, which is static once the ADF has been created and is not dependent on any connection options.

If you agree with these changes Alexa I will send out an email about the updates!

Reply:
In latest push I changed `connection_string` to `hostname` but I did not yet remove `username`. @agreenb let me know if this is something we should remove. Thanks!

Reply:
+1, hostname sounds good and question about the larger scope change in my other comment.